Add blog-post about downtream workflow (#404)
packit/packit.dev: content/docs/guide.md

```

This option is ignored in Packit running on GitHub for security reasons and **is not**
recommended either.

> Here, CI systems and packagers needs to step in and verify the proposed changes.

Reviewer: ```suggestion
Here, CI systems and packagers need to step in and verify the proposed changes.
```

Once the pull-request is merged, you can use a [`koji_build` job](/docs/configuration/#koji_build) to trigger the Koji build
for new dist-git commits. It can be configured like this:

```yaml
jobs:
- job: koji_build
  trigger: commit
  metadata:
    dist_git_branches:
      - fedora-all
```

> And to not need to wait for the successful build and create a Bodhi update yourself,

Reviewer: Parsing… _segmentation fault_
```suggestion
And if you do not wish to wait for the successful build and create the Bodhi update yourself,
```
